requirements.txt

```diff
@@ -10,7 +10,7 @@ graphviz>=0.13;platform_system!='Windows'
 seaborn>=0.11.1
 category_encoders>=2.2.2
 imbalanced-learn>=0.8.0
-pmdarima==1.8.0
+pmdarima>=1.8.1
```

Comment:
Does 1.8.1 actually work in 3.9? Let’s merge this in to fix the nightlies but wondering if we need to use >=1.8.4

Reply:
@freddyaboulton I think since I just set a lower bound, it ended up installing `1.8.4` anyways (https://github.com/alteryx/evalml/runs/4902413563?check_suite_focus=true); I can check 1.8.1 by hardcoding and see what happens? 👀

Reply:
Looks good in 3.9 even with 1.8.1!! https://github.com/alteryx/evalml/pull/3271